ci/docker/Dockerfile.maldet.leap.15

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018-2022 Intel Corporation
```

Comment:
Ditto about Docker vs. VM for this stage and this Dockerfile.

Reply:
Again, previously a change to was done to this section of master, and the merge wiped out the code for this PR,.